getting_started/workflow/assets/import_process.rst

```diff
@@ -9,10 +9,10 @@ Importing assets in Godot 3.0+
 Previously, importing assets in Godot 2.x required manual maintenance
 of a separate directory with source assets. Without doing this, it was
 impossible to specify how to convert and change import flags for
-textures, audios, scenes, etc.
+textures, audio files, scenes etc.
```

Comment:
> I'm surprised by the removal of the comma before `etc.`.

@akien-mga : etc. = et cētera
et cētera [Latin] = and [the] other(s) / and [the] rest
et = and
cētera = the others / the rest

Placing a comma before and is a relatively recent development; back in the day, it used to be a mistake.

> And our [guidelines require using the Oxford comma](http://docs.godotengine.org/en/latest/community/contributing/docs_writing_guidelines.html#use-the-oxford-comma-to-enumerate-anything) anyway so that settles it :) 

However, that does settle the matter, although with justified controversy:
"[Although the Oxford University Press uses it (hence the name), American English writers use it more often than British English writers.](https://www.grammarly.com/blog/comma-before-or/)"